tests/test_visitInfo.py

```diff
@@ -486,6 +488,67 @@ def test_str(self):
         self.assertIn("darkTime=11.02", string)
         self.assertIn("rotType=1", string)
 
+    def test_parallacticAngle(self):
+        """Check that we get the same precomputed values for parallactic angle."""
+        parallacticAngle = [141.39684140703142*degrees, 76.99982166973487*degrees]
+        for item, parAngle in zip((self.data1, self.data2), parallacticAngle):
+            visitInfo = afwImage.VisitInfo(item.exposureId,
+                                           item.exposureTime,
+                                           item.darkTime,
+                                           item.date,
+                                           item.ut1,
+                                           item.era,
+                                           item.boresightRaDec,
+                                           item.boresightAzAlt,
+                                           item.boresightAirmass,
+                                           item.boresightRotAngle,
+                                           item.rotType,
+                                           item.observatory,
+                                           item.weather,
+                                           )
+            self.assertAnglesAlmostEqual(visitInfo.getBoresightParAngle(), parAngle)
+
+    def test_parallacticAngleNorthMeridian(self):
+        """An observation on the Meridian in the Northern hemisphere has a parallactic angle of pi radians."""
```

Comment:
In the office you said it will be pi if the target is south of the telescope, 0 if north of the telescope on the meridian. I think that is clearer than "in the Norther hemisphere", especially since the hemisphere should not matter.

Also please add a test for positive or negative not on the meridian. The two tests you have now will pass regardless of the sign of the parallactic angle! I realize it may be a bit trickier to predict the correct angle, but I think it is well worth testing at least one non-trivial case, since trivial cases can all too easily hide math errors.

Also I suggest you simplify this test by only setting the VisitInfo fields that matter. VisitInfo can be constructed using keyword arguments and any of them can be omitted.

Reply:
Good points, you are correct. I will update the two meridian tests to be north and south of zenith, not in the northern or southern hemisphere.

For the test of the sign of the parallactic angle, doesn't the first test `test_parallacticAngle` catch that? There it checks the results for the coordinates of data1 and data2 against precomputed values.

Reply:
Shouldn't have underscores in Science Pipelines code.

Reply:
Okay. I was copying the style of the test immediately before the ones I added, but I suppose that one was an exception.